src/main/java/uk/gov/hmcts/reform/hmc/model/CaseCategory.java

```diff
@@ -5,15 +5,18 @@
 import uk.gov.hmcts.reform.hmc.exceptions.ValidationError;
 import uk.gov.hmcts.reform.hmc.validator.EnumPattern;
 
+import javax.validation.constraints.NotEmpty;
 import javax.validation.constraints.Size;
 
 @Data
 @NoArgsConstructor
 public class CaseCategory {
 
+    @NotEmpty(message = ValidationError.CATEGORY_TYPE_EMPTY)
```

Comment:
Can the db columns be updated to not null for all the fields to be in sync with model  class please